docs/guides/cloud/smart-orchestration/load-balancing.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: Load Balancing
+sidebar_position: 20
+---
+
+Optimize CI resources and minimize test duration by intelligently prioritizing
```

Comment:
Reading the paralyzation guide, seems like it references load balancing quite a bit. Seems like this guide should reference to it's impact on paralyzation to re-iterate how these work together.

Reply:
Added a ticket to expand this section.

Reply:
can you link?

Reply:
https://github.com/orgs/cypress-io/projects/16/views/11?pane=issue&itemId=30696671